app.js

```diff
@@ -30,68 +36,86 @@ db.once('open', function() {
 
 app.get('/getUsers', function(req, res) {
   Session.find({}, function(err, sessions) {
-    if (err) throw err;
+    if (err) console.log(`Error getting users: ${err}`);;
 
     return res.send(sessions);
   })
 })
 
+// Returns session ID and user ID (client wants both)
 app.post('/api/create-session', function(req, res) { 
-  console.log("Creating session..");
   var name = req.body.name;
   
   var users = [];
   
   users.push({
-    name: name,
-    kind: 'Participant'
+    name: name
   })
 
   var newSession = new Session ({
       users: users
   })
-
-  console.log(`New session object created: ${newSession}`)
   
   newSession.save(function(err) {
-    console.log(`Saving new session...`)
-    if (err) throw err;
+    if (err) console.log(`Error creating new session: ${err}`);
 
-    return res.json({_id: newSession._id});
-    console.log('Saved');
+    return res.json({
+      _sessionId: newSession._id,
```

Comment:
Are you using the id generated by mongodb? Might be better to use `shortId.generate()` so the URL isn't too long.

Reply:
Also just curious but why the _ in the name?

Reply:
No particular reason